recipes/mcxtrace-suite/install-files-core.bat

```diff
@@ -0,0 +1,73 @@
+set SRCDIR=%CD%\src
+
+@REM A few (silent) sanity checks that variables are set and meaningful:
+if exist %PREFIX%\ echo %PREFIX% is in place
+@rem if %errorlevel% neq 0 exit /b %errorlevel% 
+if exist %SRCDIR%\ echo %SRCDIR% is in place
+@rem if %errorlevel% neq 0 exit /b %errorlevel% 
+if exist %SRCDIR%\CMakeLists.txt echo CMakeLists.txt is in place
+@rem if %errorlevel% neq 0 exit /b %errorlevel% 
+
+verify other 2> nul
+setlocal eneableextensions
+if errorlevel 1 echo Unable to enable extensions 
+@rem if %errorlevel% neq 0 exit /b %errorlevel% 
+if defined PKG_VERSION echo version %PGK_VERSION% 
+@rem if %errorlevel% neq 0 exit /b %errorlevel% 
+endlocal
+
+mkdir %CD%\build_mcxtrace_core
+set BLDDIR=%CD%\build_mcxtrace_core
+
+cd %BLDDIR%
+
+@REM Configure.
+cmake ^
+    -DCMAKE_INSTALL_PREFIX=%PREFIX% ^
+    -S %SRCDIR% ^
+    -G "NMake Makefiles" ^
+    -DMCVERSION=%PKG_VERSION% ^
+    -DMCCODE_BUILD_CONDA_PKG=ON ^
+    -DCMAKE_INSTALL_LIBDIR=lib ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DBUILD_MCXTRACE=ON ^
+    -DMCCODE_USE_LEGACY_DESTINATIONS=OFF ^
+    -DBUILD_TOOLS=ON ^
+    -DENABLE_COMPONENTS=ON ^
+    -DENSURE_MCPL=OFF ^
+    -DENSURE_NCRYSTAL=OFF ^
+    -DENABLE_CIF2HKL=OFF ^
+    -DENABLE_NEUTRONICS=OFF ^
+    -DBUILD_SHARED_LIBS=ON
+
+@REM ^	DCMAKE_C_COMPILER=gcc.exe
+
+	
+cmake --build . --config Release
+
```

Comment:
Is there a `make check` or `ctest` that we can run here?

Reply:
@ocefpaf everything of this sort is thoroughly tested in test_pkg.py script in fact: Existence of certain files and binaries, test runs of c-code generator and a simulation run.

Reply:
Is that an OK approach? Everything is run in the same workflow and we fail in case anything is missing / exits in error etc.

Reply:
That is OK but, if there is a `make check`/`ctest`, why not run it here? Do you know if there are any?

Reply:
There is currently no test like that implemented

Reply:
I take it we can live with the install-time based tests from `test_pkg.py`? (To remember the recommendationI can create an issue upstream for a more standard `make check` but will likely not have bandwidth to do anything else about it for a while...)

Reply:
https://github.com/McStasMcXtrace/McCode/issues/1576

Reply:
@ocefpaf I take the liberty of marking this as resolved as we have no built in "make test" upstream, but I have taken note and written a feature-request for it. :)